stream_alert_cli/terraform/generate.py

```diff
@@ -21,6 +21,7 @@
 from stream_alert.shared import metrics
 from stream_alert_cli.logger import LOGGER_CLI
 from stream_alert_cli.terraform._common import (
+    DEFAULT_SNS_MONITORING_TOPIC,
```

Comment:
If you import ` DEFAULT_SNS_MONITORING_TOPIC` here, please remove line 38 where redefine it.

Reply:
Dang - great catch!

Reply:
Sorry, this file is outdated. I was looking at an old commit.